lib/logger/lib/logger.ex

```diff
@@ -489,8 +489,13 @@ defmodule Logger do
           level: min_level, utc_log: utc_log?} = Logger.Config.__data__
 
         if compare_levels(level, min_level) != :lt do
-          truncated = truncate(chardata_or_fn, truncate)
-          metadata = [pid: self()] ++ Keyword.merge(pdict, metadata)
+          {message, fn_metdata} = normalize_message(chardata_or_fn)
```

Comment:
Since we touching these lines, I think it is good opportunity to change `_fn` –> `_fun`, we usually use the latter one.